src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -362,11 +370,17 @@ export const selectPortfolioAssetIdsByAccountIdExcludeFeeAsset = createSelector(
   selectPortfolioAssetAccountBalancesSortedFiat,
```

Comment:
this selector is pretty expensive, do you want to use the deep equal selector here, as the output array is cheap to serialize